nautobot/extras/migrations/0033_alter_tag_content_types.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.2.12 on 2022-04-05 02:26
+
+from django.db import migrations, models
+import nautobot.extras.utils
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("contenttypes", "0002_remove_content_type_name"),
+        ("extras", "0032_tag_content_types_data_migration"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="tag",
+            name="content_types",
+            field=models.ManyToManyField(
+                blank=True,
```

Comment:
In the final migration, shouldn't this be a non-blank-allowed field?

Reply:
Yes it should not be blank/nullable!